android/app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
+    "project_info": {
+      "project_number": "663889381642",
+      "project_id": "firebase-leapfrogger"
+    },
+    "client": [
+      {
+        "client_info": {
+          "mobilesdk_app_id": "1:663889381642:android:3f78a7c51140174d",
+          "android_client_info": {
+            "package_name": "com.leapfroggers"
+          }
+        },
+        "oauth_client": [
+          {
+            "client_id": "663889381642-1o6p6su9bmrrf45pv96bvtpkdoqguo66.apps.googleusercontent.com",
+            "client_type": 1,
+            "android_info": {
+              "package_name": "com.leapfroggers",
```

Comment:
@kusalshrestha we normally don't push this type of file to github.

Reply:
🆗 @surhidamatya